packages/vscode-extension/CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## 4.0.0 - May 24, 2022
+
+Major version for Teams Toolkit with new features to support more Teams app scenario like notification bot and commend bot. What's more, this version adds support to extend Teams app across M365 platform like Office 365 and Outlook.
+
+New Features:
+
+- User can create more business-oriented Teams app template using Teams Toolkit. For example, user can now create not only Bot app but also Notification Bot or Command Bot. User can see more options are there to choose when create a new Teams app.
+- User can create Teams app that can launch and preview in other M365 platform like Office 365 and Outlook. The options are offered when user create a new Teams app.
```

Comment:
Nit: maybe we do not need to capitalize words like `Notification Bot` and `Command Bot`?

Reply:
changed